events.csv

```diff
@@ -31,4 +31,6 @@ Makers Hour, 2025-12-11, 20:00:00, 2025-12-11, 21:00:00, "Chat and question sess
 Makers Hour, 2025-12-18, 20:00:00, 2025-12-18, 21:00:00, "Chat and question session on mastodon", "https://cupoftea.social/@MakersHour", , 
 Makers Hour, 2025-12-25, 20:00:00, 2025-12-25, 21:00:00, "Chat and question session on mastodon", "https://cupoftea.social/@MakersHour", , 
 
-Gauss Fest, 2025-09-07, 10:00:00, 2025-09-07, 16:00:00, "Discover the fascinating world of electricity at Gaussfest! A public demonstration of Tesla Coils, High Voltage Multiplies, Van De Graffs, and Violet Ray Machines. Experience indoor lightning and magnificent light shows!", https://www.extremeelectronics.co.uk/the-gaussfest/, 51.488887, -0.290400
\ No newline at end of file
+Gauss Fest, 2025-09-07, 10:00:00, 2025-09-07, 16:00:00, "Discover the fascinating world of electricity at Gaussfest! A public demonstration of Tesla Coils, High Voltage Multiplies, Van De Graffs, and Violet Ray Machines. Experience indoor lightning and magnificent light shows!", https://www.extremeelectronics.co.uk/the-gaussfest/, 51.488887, -0.290400
+Makers Central, 2026-05-16, 10:00:00, 2026-05-16, 17:30:00, "A fun family event for creators, makers and hobbyists!", "https://www.makerscentral.co.uk/", 52.4512, -1.7212
```

Comment:
The calendar system will handle multi day events, this could be one entry with the end date set to the 17th

Reply:
oh wait they have different end times. That's actually more useful this way around.